V/Valhalla/build_tarballs.jl

```diff
@@ -58,25 +83,26 @@ install_license ../LICENSE.md
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-# Windows is blocked until pkg-config issues are figured out (https://github.com/valhalla/valhalla/issues/3931)
-platforms = supported_platforms(; exclude=Sys.iswindows)
+platforms = supported_platforms()
 platforms = expand_cxxstring_abis(platforms)
 
 # The products that we will ensure are always built
 products = Product[
     LibraryProduct("libvalhalla", :libvalhalla),
+    ExecutableProduct("valhalla_service", :valhalla_service),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("boost_jll"; compat="=1.76.0")
-    Dependency("GEOS_jll")
-    Dependency("LibCURL_jll")
+    HostBuildDependency(PackageSpec(name="protoc_jll", version=v"105.29.3"))
+    Dependency("boost_jll"; compat="=1.87.0")
+    Dependency("GEOS_jll"; compat="3.13.1")
+    Dependency("LibCURL_jll"; compat="7.73,8")
     Dependency("Lz4_jll")
-    Dependency("protoc_jll")
-    HostBuildDependency("protoc_jll")
+    Dependency("protoc_jll"; compat="105.29.3")
     Dependency("Zlib_jll")
+    Dependency("OpenSSL_jll"; compat="3.0.16")
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"6")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"11.1", clang_use_lld=true)
```

Comment:
`clang_use_lld` already defaults to `true`: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/c925b8466bf77d6fba957ba6ddf2a71a3449e35d/src/Runner.jl#L1405-L1406